msgboard/templates/msgboard/base_board.html

```diff
@@ -0,0 +1,42 @@
+<html>
+<head>
```

Comment:
perhaps use bootstrap instead of writing your own CSS

Reply:
Thanks i will deal with in new branch :+1:  thanks